src/test/smoke/pages/steps.js

```diff
@@ -0,0 +1,36 @@
+const requireDirectory = require('require-directory');
+
+const steps = requireDirectory(module);
+
+const actions = {};
+
+function setActorActions(data) {
+  // eslint-disable-next-line no-console
+  // console.log('Setting Action actions for data : '+JSON.stringify(data));
```

Comment:
Did you mean to leave these console logs commented in the code?

Reply:
this is common code for end to end tests. I can remove this code as it looks like not mandatory